junit5/src/main/java/org/jboss/weld/junit5/auto/ClassScanning.java

```diff
@@ -255,14 +253,16 @@ private static Extension createExtension(Class<? extends Extension> clazz) {
         }
     }
 
+    private static boolean isBeanParameterAnnotation(Annotation ann) {
+        return isAnnotated(ann.annotationType(), Qualifier.class) ||
+                isAnnotated(ann.annotationType(), Scope.class) ||
```

Comment:
If I get you right here, you use this to check method parameters.
In that case, the parameters need to have qualifiers to be considered beans for injection. You will never have a scope there.